docs/demo/scripts/kind/Makefile

```diff
@@ -122,9 +127,49 @@ kind-create: temp-dir kind kind-delete ## Create the Kind cluster
 	$(KIND) create cluster --config $(TEMP_DIR)/kind-config.yaml ; \
 	rm $(TEMP_DIR)/kind-config.yaml
 
+.PHONY: whereabouts
+whereabouts: temp-dir ## Install whereabouts IPAM CNI plugin
+	git clone https://github.com/k8snetworkplumbingwg/whereabouts $(TEMP_DIR)/whereabouts && \
+	kubectl apply \
+	-f $(TEMP_DIR)/whereabouts/doc/crds/daemonset-install.yaml \
+	-f $(TEMP_DIR)/whereabouts/doc/crds/whereabouts.cni.cncf.io_ippools.yaml \
+	-f $(TEMP_DIR)/whereabouts/doc/crds/whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml && \
+	kubectl rollout status daemonset whereabouts -n kube-system --timeout 300s || \
+	{ echo >&2 "Error: Failed to install Whereabouts"; rm -fr $(TEMP_DIR)/whereabouts; exit 1; } ; \
+	rm -fr $(TEMP_DIR)/whereabouts
```

Comment:
Direct links could be used:
https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/master/doc/crds/daemonset-install.yaml
https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/master/doc/crds/whereabouts.cni.cncf.io_ippools.yaml
https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/master/doc/crds/whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml